package.json

```diff
@@ -128,6 +128,7 @@
     "autoprefixer": "^10.4.14",
     "cypress": "^13.5.0",
     "cypress-localstorage-commands": "^2.2.3",
+    "dotenv": "^16.4.5",
```

Comment:
Why was this added? Context?

Reply:
Actually, I don't know why using env variables wasn't working in vite.config.mts, when I used "import 'dotenv/config'", it worked. That's the reason.

Reply:
that was onw of the reasons why I was unable to connect to the local care backend.

Reply:
We are using [direnv](https://direnv.net/docs/installation.html) in this repository to load environment variables as configured [here](https://github.com/coronasafe/care_fe/blob/develop/.envrc)

Reply:
You could also connect to your local backend using: 
```
CARE_API=https://127.0.0.1:8000/ bun run dev
```

Reply:
Ok, will try that.

Reply:
CARE_API=https://127.0.0.1:9000/ bun run dev
I added the above line in my .env, but unable to connect to the local backend which is running at http://localhost:9000/.

Reply:
What should I do now?

Reply:
Also, after removing dotenv, when I am using `process.env.CARE_API`, in `vite.config.mts`, its not working, but working when I am explicitly mentioning https://127.0.0.1:9000/.

Reply:
> CARE_API=[127.0.0.1:9000](https://127.0.0.1:9000/) bun run dev I added the above line in my .env, but unable to connect to the local backend which is running at [http://localhost:9000](http://localhost:9000/).

You shouldn't add that to the `.env`
Instead use that to run in the terminal directly.

Reply:
OK, if that's the thing, will try to do it.

Reply:
Hey, I have pushed some changes, could you please review them and suggest if any modification is required.

Reply:
Also all the tests are now passing successfully.